src/models/split_transfers.py

```diff
@@ -99,39 +115,44 @@ def _process_token_transfers(
                     # Reinsert since there is still an amount owed.
                     self.overdrafts[solver] = overdraft
                     continue
-            if solver in cow_redirects:
-                # Redirect COW rewards to reward target specific by VouchRegistry
-                redirect_address = cow_redirects[solver].reward_target
-                log_saver.print(
-                    f"Redirecting solver {solver} COW tokens "
-                    f"({transfer.amount}) to {redirect_address}",
-                    category=Category.REDIRECT,
-                )
-                transfer.receiver = redirect_address
+            transfer.redirect_to(redirect_map, self.log_saver)
             self.cow_transfers.append(transfer)
+        # We do not need to worry about any controversy between overdraft
+        # and positive slippage adjustments, because positive/negative slippage
+        # is disjoint between solvers.
+        while positive_slippage:
+            slippage_transfer = Transfer.from_slippage(positive_slippage.pop())
```

Comment:
For consistency with the other method, should we assert here that slippage is positive?

Reply:
Done in https://github.com/cowprotocol/solver-rewards/pull/163/commits/4057c8b4dad10cc6a1ef312dcef570114772543f